charts/vald/values.yaml

```diff
@@ -4430,3 +4430,251 @@ initializer:
+    # rebalancer.clusterRole.enabled -- creates clusterRole resource
+    enabled: true
+    # @schema {"name": "rebalancer.clusterRole.name", "type": "string"}
+    # rebalancer.clusterRole.name -- name of clusterRole
+    name: rebalancer
+  # @schema {"name": "rebalancer.clusterRoleBinding", "type": "object"}
+  clusterRoleBinding:
+    # @schema {"name": "rebalancer.clusterRoleBinding.enabled", "type": "boolean"}
+    # rebalancer.clusterRoleBinding.enabled -- creates clusterRoleBinding resource
+    enabled: true
+    # @schema {"name": "rebalancer.clusterRoleBinding.name", "type": "string"}
+    # rebalancer.clusterRoleBinding.name -- name of clusterRoleBinding
+    name: rebalancer
+  # @schema {"name": "rebalancer.serviceAccount", "type": "object"}
+  serviceAccount:
+    # @schema {"name": "rebalancer.serviceAccount.enabled", "type": "boolean"}
+    # rebalancer.serviceAccount.enabled -- creates service account
+    enabled: true
+    # @schema {"name": "rebalancer.serviceAccount.name", "type": "string"}
+    # rebalancer.serviceAccount.name -- name of service account
+    name: vald
+  # @schema {"name": "rebalancer.rebalancer", "type": "object"}
+  rebalancer:
+    # @schema {"name": "rebalancer.rebalancer.pod_name", "type": "string"}
+    # rebalancer.rebalancer.pod_name -- pod name of rebalancer stroage controller
+    pod_name: _MY_POD_NAME_
+    # @schema {"name": "rebalancer.rebalancer.pod_namespace", "type": "string"}
+    # rebalancer.rebalancer.pod_namespace -- pod namespace of rebalancer stroage controller
+    pod_namespace: _MY_POD_NAMESPACE_
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_name", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_name -- name of rebalancer job
+    rebalance_job_name: vald-rebalancer-storage-job
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_namespace", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_namespace -- namespace of rebalancer job
+    rebalance_job_namespance: _MY_POD_NAMESPACE_
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_template_key", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_template_key -- template key of rebalancer job
+    rebalance_job_template_key: job.tpl
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_image", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_image -- container image of rebalancer job
+    rebalance_job_image: hello-world
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_configmap_name", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_configmap_name -- configmap name of rebalancer job
+    rebalance_job_configmap_name: vald-rebalancer-storage-job-config
+    # @schema {"name": "rebalancer.rebalancer.rebalance_job_configmap_namespace", "type": "string"}
+    # rebalancer.rebalancer.rebalance_job_configmap_namespace -- configmap namespace of rebalancer job
+    rebalance_job_configmap_namespace: _MY_POD_NAMESPACE_
```

Comment:
Is there any reason that `rebalance_job_configmap_namespace` is different from `rebalance_job_namespace`? I think they must be same.

Reply:
At begging the implementation, we thought the namespace of rebalance job can be chosen by user definition ignoring good or not.
but now, it should be needed, so I agree w/u

Reply:
@rinx

Reply:
fixed